sdk/mgmtcommon/AppAuthentication/Azure.Services.AppAuthentication/Helpers/EnvironmentHelper.cs

```diff
@@ -31,6 +31,14 @@ public static string SystemDirectory
                 return sb.ToString();
             }
         }
+#else
+        public static string SystemDirectory
```

Comment:
Suggestion: #if/def inside the property getter -- that way it is less likely that the 2 will diverge in the future.

Reply:
Good suggestion, done.